subtests/docker_cli/meta/meta.py

```diff
@@ -0,0 +1,62 @@
+r"""
+Summary
+-------
+
+Meta-tests: docker packaging, installation, setup. Nothing here
+actually runs docker.
+
+Operational Summary
+-------------------
+
+#. Run 'file' against docker binaries, make sure output includes 'not stripped'
+"""
+
+import os
+import magic
+from dockertest import subtest
+from dockertest.subtest import SubSubtest
+
+
+class meta(subtest.SubSubtestCaller):
+
+    """ SubSubtest caller """
+
+
+class meta_base(SubSubtest):
+    pass
+
+
+class debuginfo_present(meta_base):
+    """
+    bz1280068 - make sure docker binary is shipped with debug info.
+    We simply use libmagic to check for us. Note that this may fail
+    if/when it becomes possible to ship debuginfo as a separate rpm.
+    """
+
+    def initialize(self):
+        super(debuginfo_present, self).initialize()
+        # The python-magic package that ships with RHEL is clunky, obsolete,
+        # undocumented, and bears no relation to any currently known magic.py
+        # in other distros or via pip. Refer to:
+        #     https://stackoverflow.com/questions/25286176
+        self.sub_stuff['magic'] = magic.open(magic.MAGIC_NONE)
+        self.sub_stuff['magic'].load()
+
+    def run_once(self):
+        super(debuginfo_present, self).run_once()
+        found_binary = False
+        # Another way to search for DWARF might be:
+        #    readelf -S /usr/bin/docker-current |grep -i debug_info
+        for binfile in ['docker-current', 'docker-latest']:
+            path = os.path.join('/usr/bin', binfile)
+            if os.path.exists(path):
+                found_binary = True
```

Comment:
Second time through the loop (``binfile == "docker-latest"``), ``found_binary`` will overwrite the prior iteration value.  i.e. ``docker-current`` is not found but ``docker-latest`` is found.  Probably okay to just get rid of the ``os.path.exists()`` check and allow ``self.sub_stuff['magic'].file(path)`` to fail.  Disadvantage is that failure on the first item would prevent testing of the second.

An alternative idea (to always test both) could be to use test-iterations as list indexes into the binfile names.  IIRC, all you need to do is set a class attribute ``iterations = 2``, and another for ``binfiles = ['docker-current', 'docker-latest']``.  Then, in your ``run_once()``, set ``binfile = self.binfiles[self.iteration - 1]`` (or something like that).